lib/control-app.js

```diff
@@ -30,9 +31,13 @@ module.exports = function(config) {
         fullPaths: true
     });
 
-    files = files.map(function(file) {
-        return path.resolve(file);
+    var expandedFiles = [];
+    files.forEach(function (arg) {
```

Comment:
have no strong preference there, using a tested lib seems good :)

Once you have the code how you like, squash the commits and ping me for the PR again